apps/admin/backend/src/cvr_files.ts

```diff
@@ -592,6 +606,59 @@ export async function addCastVoteRecordReport({
         ).BatchLabel,
       });
 
+      // Add referenced images and layouts to the cast vote record
+      if (cvr.BallotImage) {
+        // Convention is that we always have two entries in the BallotImage
+        // array, allowing us to indicate front and back via array index.
+        assert(cvr.BallotImage.length === 2);
```

Comment:
As long as the write-in entries nested in the CDF CVR reference the correct image which is in the CVR's `BallotImage` field, I'm not sure if this convention is necessary in the long run. But for now, it is a useful simpllification.